packages/reason-bindings/src/FlopFlip_ReactBroadcast__ToggleFeature.re

```diff
@@ -0,0 +1,40 @@
+open FlopFlip__Types;
+
+module Subscriber = {
+  [@bs.module "react-broadcast"]
+  external reactClass : ReasonReact.reactClass = "Subscriber";
```

Comment:
Do we have to bind this component as it‘s used by ToggleFeature?

Reply:
Well yes, the ToggleFeature implementation here is different from the original one.